package.json

```diff
@@ -30,7 +30,7 @@
     "express": "^4.16.4",
     "get-folder-size": "^2.0.0",
     "ipfs": "^0.33.1",
-    "orbit-db": "^0.19.9",
+    "orbit-db": "git://github.com/orbitdb/orbit-db.git#9d674766a54c7b949eca0d7e5ddc26b5486717cd",
```

Comment:
which orbit feature/fix is this for specifically? 

need to do the same for scaling, to get orbitdb cache options

Reply:
It's for the `determineAddress` function.

Reply:
ok cool, just checked this ref used the cache option too, so we have everything we need for new arch too

Reply:
Yup, I made sure as well :)

Reply:
also any concern using a different version of orbitdb in pinning than 3boxjs ? 19.9 was released last july, but doesn't look like to many major code changes up to this commit

Reply:
Nope, the PR for `determineAddress` should be the only change. 
However let's make sure to test this once it's deployed to develop :)